test/itkVkMultiResolutionPyramidImageFilterFactoryTest.cxx

```diff
@@ -0,0 +1,56 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include <complex>
+#include <string>
+
+#include "itkMultiResolutionPyramidImageFilter.h"
+#include "itkVkMultiResolutionPyramidImageFilter.h"
+
+#include "itkVkMultiResolutionPyramidImageFilterFactory.h"
+#include "itkTestingMacros.h"
+
+// Verify MultiResolutionPyramidImageFilter can be overriden
+// with spatial+FFT implementation through object factory override
+
+int
+itkVkMultiResolutionPyramidImageFilterFactoryTest(int, char *[])
+{
+  using PixelType = double;
+  const unsigned int Dimension{ 2 };
```

Comment:
`constexpr` would be better since it is used for a template parameter.

Reply:
Fixed